apps/web/components/booking/BookingDescriptionPayment.tsx

```diff
@@ -5,8 +5,7 @@ import { FiCreditCard } from "@calcom/ui/components/icon";
 
 const BookingDescriptionPayment = (props: { eventType: Parameters<typeof getPaymentAppData>[0] }) => {
   const paymentAppData = getPaymentAppData(props.eventType);
-
-  if (!paymentAppData || paymentAppData.price > 0) return null;
+  if (!paymentAppData || paymentAppData.price <= 0) return null;
```

Comment:
@zomars FYI, have changed this check as it was rendering only when payment isn't required..